frontend/app/src/components/AppView/AppView.tsx

```diff
@@ -210,14 +208,6 @@ function AppView(props: AppViewProps): ReactElement {
         {(!embedded || showFooter) && (
```

Comment:
Do we not need this anymore then?

Reply:
Hmm, this is a good question. Probably not. I will remove it and do some testing if I find something unexpected.

Reply:
Ok, removing the spacer seems to be fine 👍 But I will wait until Monday to merge so that we have enough time to find any issue with this potentially.